debian/jessie/foreman/rules

```diff
@@ -26,3 +26,9 @@ build:
 	/bin/sed -ri '1sXenv rubyXenv foreman-rubyX' bin/*
 	/bin/sed -ri 's~^BUNDLER_CMD=""~BUNDLER_CMD="/usr/bin/foreman-ruby /usr/bin/bundle exec"~' script/foreman-rake
 	/bin/mkdir -p .ssh
+	# Ensure backwards compatiblity by having dynflowd aliased to foreman-tasks
+	/bin/mv /usr/sbin/dynflow-executor /usr/sbin/dynflowd
```

Comment:
See my fixes in https://github.com/theforeman/foreman-packaging/pull/1957. This is the build step, not the install step. I fixed it with ```mv bin/dynflow-executor script/dynflowd``` and set the right properties in foreman.install. The symlink is solved via the links file.

Reply:
I realize that, but that's not the reason this PR is failing. I tried to make some changes in `stretch` (https://github.com/theforeman/foreman-packaging/pull/1959#discussion_r152921779) and when stretch builds fine, I will propagate the changes to the rest.